mautic/mautic
# Add new column Source of created contact to Page hit report

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added new column  Source of created contact to Page hits records. 
This column is related to lead_event_log table and allow reports page hits during contact was created.  
You can also use COUNT aggregate function of the column, what return number of contacts created from each page hit.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. You need create several page hits from incognito window for example, what will be first tracked users events. These hits should be marked to contact timeline as Contact created from source

![image](https://user-images.githubusercontent.com/462477/64681640-5fd1d400-d480-11e9-8844-ef8c8e54abbf.png)


3. Then you can create page hits report  with new columns. Also you ca use it in filter and in aggregated function. Your report should looks like

![image](https://user-images.githubusercontent.com/462477/64681745-90197280-d480-11e9-9b70-f1a48c5bf068.png)

4. Test If your results looks like you expect
